src/theme/components/styledList.ts

```diff
@@ -0,0 +1,25 @@
+import { baseListStyles, baseUnorderedStyles } from "./list";
+
+const StyledList = {
+  baseStyle: ({ textSize = "default" }) => ({
+    fontSize: `text.${textSize}`,
+    ...baseListStyles(false, true),
+    ...baseUnorderedStyles(true),
+  }),
+  variants: {
+    capped: {
+      li: {
+        borderLeft: "1px solid",
```

Comment:
Should we use border-inline-start rather than left per your multilingual talk yesterday?

https://developer.mozilla.org/en-US/docs/Web/CSS/border-inline-start

Reply:
There was something that did not work when I tried that initially. Maybe I'll have to do a workaround but you're right that we shouldn't use "left".